### PR TITLE
RAWR Work Branch Clean Up

### PR DESCRIPTION
**Pull request by ayushchakra**
This PR is intended to clean up the progress that has been made in the RAWR work branch, leaving it in a state where it can be merged into the master branch.
Outstanding Action Items:
- Refactor shift metric to live in data models
- Complete documentation
- Remove unnecessary artifacts

As a follow up to this PR, a suite of GitHub Actions should be added to this repo to check the following:

- Run Pylint and block merges until all issues are resolved
- Basic sweep workflow to ensure no broken code
- Require at least one review before enabling merge
- Not an action but might be nice to bringup ruff (linter) and develop a github pre-commit hook to run it

### CODE REVIEW

File: .vscode/settings.json
Comment (richardli03) [inline .vscode/settings.json:1]: I believe that .vscode as a folder should be hidden right? Does this linter setting need to be here? I'm also confused why the .gitignore doesn't contain this entire folder.

File: archive/plot_graph.py
Comment (richardli03) [inline archive/plot_graph.py:1]: Gonna check off all of these archive files without reading them, assuming it's fine.

**Review by richardli03 — APPROVED** [review]
Seems like most of this was the linter doing its fixes! Thank you for your work :) . Only clarification is why the .vscode file is getting pushed.

File: .vscode/settings.json
Comment (ayushchakra) [reply .vscode/settings.json:1]: Ah my bad yea - adding it to the .gitignore

**Pull request merged**